front/src/features/unified/Form.js

```diff
@@ -21,13 +21,17 @@ const Form = ({
   const dispatch = useDispatch();
   const location = useLocation();
 
-  const updateResult = (
-    content,
-    { id, campaign, character, bbMessage } = {}
-  ) => {
-    setResult({ content, id, bbMessage });
-    dispatch(addCampaign(campaign));
-    dispatch(addCharacter(character));
+  const updateResult = (data) => {
+    let results = [];
+    (Array.isArray(data) ? data : [data]).forEach(
+      ({ character, campaign, ...result }) => {
+        dispatch(addCampaign(campaign));
+        dispatch(addCharacter(character));
```

Comment:
One day, someone will rework the addCampaign/addCharacter thing (used so that the autocomplete is always up to date without a page) so it's not so much of a hassle every time.
One day.